recipe/patches/0017-point-include-paths-to-PREFIX-include.patch

```diff
@@ -0,0 +1,33 @@
+From 7c4094fe8edfd01d72b664c91a60cc14bb5a4dbe Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Thu, 23 Jan 2025 22:58:14 +1100
+Subject: [PATCH 17/17] point include paths to $PREFIX/include
+
+---
+ torch/utils/cpp_extension.py | 9 +++++++++
+ 1 file changed, 9 insertions(+)
+
+diff --git a/torch/utils/cpp_extension.py b/torch/utils/cpp_extension.py
+index 3f584ef5598..4210f62b6db 100644
+--- a/torch/utils/cpp_extension.py
++++ b/torch/utils/cpp_extension.py
+@@ -1155,10 +1155,19 @@ def include_paths(cuda: bool = False) -> List[str]:
+         A list of include path strings.
+     """
+     lib_include = os.path.join(_TORCH_PATH, 'include')
++    if os.environ.get("CONDA_BUILD", None) is not None:
++        pieces = [os.environ["PREFIX"]] + IS_WINDOWS * ["Library"] + ["include"]
++        lib_include = os.path.join(*pieces)
++    elif os.environ.get("CONDA_PREFIX", None) is not None:
++        pieces = [os.environ["CONDA_PREFIX"]] + IS_WINDOWS * ["Library"] + ["include"]
```

Comment:
I wish we wouldn't have to be explicit about that `"Library"`.

Reply:
Perhaps one day we can get rid of it... https://github.com/conda-forge/conda-forge.github.io/issues/2295